ttkwidgets/frames/toggledframe.py

```diff
@@ -2,13 +2,18 @@
 Author: RedFantom
 License: GNU GPLv3
 Source: This repository
+
+Improved by rdbende
 """
+
 import tkinter as tk
+from pathlib import Path
 from tkinter import ttk
-import os
-from PIL import Image, ImageTk
+
 from ttkwidgets.utilities import get_assets_directory
 
+assets_dir = Path(get_assets_directory())
```

Comment:
I like the use of `pathlib`, but why does it have to be a global variable?

Reply:
Well, I thought I might need it elsewhere, but it turns out I didn't.